google/cloud/bigquery/opentelemetry_tracing.py

```diff
@@ -64,8 +59,18 @@ def create_span(name, attributes=None, client=None, job_ref=None):
                     Raised if a span could not be yielded or issue with call to
                     OpenTelemetry.
             """
+    global WARNED_OPENTELEMETRY
     final_attributes = _get_final_span_attributes(attributes, client, job_ref)
     if not HAS_OPENTELEMETRY:
+        if not WARNED_OPENTELEMETRY:
+            logger.debug(
+                "This service is instrumented using OpenTelemetry. "
+                "OpenTelemetry could not be imported; please "
+                "add opentelemetry-api and opentelemetry-instrumentation "
+                "packages in order to get BigQuery Tracing data."
+            )
+            WARNED_OPENTELEMETRY = True
```

Comment:
(nit) It's a bit unexpected to change a "constant", perhaps rename it? Also, since it's an internal state-keeping flag that users should neither know about nor modify it, we should probably mark it as such (e.g. `_warned_telemetry`)

Reply:
Good call. I've renamed in the latest commit.